Makefile

```diff
@@ -0,0 +1,121 @@
+DEFAULT_GOAL := help
+
+# Ensure everything works even if GOPATH is not set, which is often the case.
+# The `go env GOPATH` will work for all cases for Go 1.8+.
+GOPATH            ?= $(shell go env GOPATH)
+
+TMP_GOPATH        ?= /tmp/thanos-go
+GOBIN             ?= $(firstword $(subst :, ,${GOPATH}))/bin
+GOPROXY           ?= https://proxy.golang.org
+GIT               ?= $(shell which git)
+
+# Version from 12.12.2019
+GOJSONTOYAML_VERSION    ?= bf2969bbd742d117a9524b859fb417fefb67565d
+GOJSONTOYAML            ?= $(GOBIN)/gojsontoyaml-$(GOJSONTOYAML_VERSION)
+
+# Version from 9.02.2020. We use https://github.com/google/go-jsonnet for ease of building.
+GO_JSONNET_VERSION         ?= 31b9ace0f65a5a5717ed9521b57d8d9e18f24070
+GO_JSONNET                 ?= $(GOBIN)/jsonnet-$(GO_JSONNET_VERSION)
```

Comment:
I actually use the C++ version most of the time as that one has `jsonnetfmt` which the Go version still has not... :roll_eyes: :expressionless:

Reply:
Ok, but we use Go for a reason. 

I think we can install one from release page easily if you prefer this, exactly as CI was doing.

Reply:
Or we can only use c++ version of `jsonnetfmt` and keep `go-jsonnet` one for other uses.

Reply:
You could find this useful:
https://github.com/observatorium/observatorium/blob/e009b4adfef6b4134073310f8e84bb3aa4215cde/Makefile#L127-L132